Gopkg.toml

```diff
@@ -11,7 +11,7 @@
   name = "github.com/bitrise-io/go-utils"
 
 [[constraint]]
-  branch = "master"
+  branch = "TOOL-1126-swiftpm-cache"
```

Comment:
merge this PR first: http://github.com/bitrise-io/go-xcode/pull/85
then update this to lock for the master branch